package.json

```diff
@@ -27,6 +27,7 @@
     "prepare": "husky"
   },
   "dependencies": {
+    "chalk": "^5.3.0",
```

Comment:
It was a bit hard to parse the terminal output, so I brought this in to make it more human readable.

Reply:
I ended up pinning this to `5.3.0` in [4326508](https://github.com/artsy/quantum/pull/6/commits/4326508460fa201cf232ab9c19f93dc0ff23f2d8)since that version is already in our `node_modules` as a dependency. So this way we're not actually bringing in another package.